packages/migration/scripts/utils/updateTemplates.js

```diff
@@ -23,7 +23,6 @@ import { join, resolve } from 'path';
 /**
  * Internal dependencies
  */
-// eslint-disable-next-line import/no-unresolved
```

Comment:
This line is required. It will be flagged as unneeded once you run a migration, which is misleading, but that doesn't mean it can be removed.